corehq/toggles/__init__.py

```diff
@@ -1644,6 +1644,21 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN]
 )
 
+
+DISABLE_MOBILE_ENDPOINTS = StaticToggle(
+    'disable_mobile_endpoints',
+    'Disable mobile endpoints for form submissions and restores',
+    TAG_INTERNAL,
```

Comment:
Would `TAG_CUSTOM` not better represent this?

Reply:
I checked with Mary. She feels that "custom" tags end up getting used more than one would expect. For now she would prefer to use the "internal" tag to signal that domains wanting to use it should get confirmation that this is right for them.